examples/vanilla-ts-esm/public/mux-uploader-full.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Basic &lt;mux-uploader/&gt; example</title>
+    <script type="module" src="./dist/mux-uploader.js"></script>
+    <style>
+      body {
+        margin: 0;
+        background: #f8f8f8;
+      }
+
+      .direct-upload-url {
+        padding: 8px 12px;
+        width: 400px;
+      }
+
+      .container {
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        font-family: Arial;
+        height: 90vh;
+      }
+
+      header {
+        margin-bottom: 100px;
+      }
+
+      header > h1 {
+        font-size: 5vw;
+        line-height: 6vw;
+        text-align: left;
+        max-width: 45vw;
+        z-index: 10;
+        position: relative;
+      }
+
+      h2 {
+        font-size: 3vw;
+        line-height: 4vw;
+        text-align: left;
+        max-width: 45vw;
+        z-index: 10;
+        position: relative;
+      }
+
+      input {
+        max-width: 45vw;
+        margin-bottom: 50px;
+        z-index: 10;
+        position: relative;
+      }
+
+      main {
+        box-sizing: border-box;
+        padding: 20px;
+        height: 100%;
+        width: 100%;
+      }
+
+      .button-container {
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+      }
+
+      .second-button {
+        align-self: flex-end;
+        margin-top: 20px;
+        z-index: 1;
+        position: relative;
+      }
+
+      footer {
+        box-sizing: border-box;
+        width: 100%;
+        padding: 20px;
+        z-index: 10;
+        position: relative;
+      }
+      
+      mux-uploader {
+        align-self: flex-end;
+
+        --button-border-radius: 40px;
+      }
+
+      mux-uploader[upload-in-progress] {
+        min-width: 250px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container">
+        <main>
+            <header>
+              <h1>Add a video.</h1>
+              <h1>Get a shareable link to stream it.</h1>
+            </header>
+            <h2>Enter your upload GCS url:</h1>
+            <input type="text" class="direct-upload-url" placeholder="https://storage.googleapis.com/..." />
+            <div class="button-container">
+              <mux-uploader type="bar" id="uploader-controller">
+                <mux-uploader-drop slot="dropzone" text="Upload to stream.new" fullScreen enableOverlay enableStatus></mux-uploader-drop>
```

Comment:
(blocking): for attributes prefer to use lowercase;
- `fullscreen`
- `enable-overlay` or just `overlay` since it's a boolean attribute
- `enable-status` or `status`